### ISSUE TITLE
Widget library dialog uses clickable divs instead of buttons

### ISSUE THREAD

**Issue by msnyder-msft** [microsoft/iot-cardboard-js#245]
**Describe the bug**
The list items in the widget library dialog are clickable divs instead of buttons. 
This is an accessibility issue for screen readers at least

**To Reproduce**
1. Go to Behaviors list
2. Click on a behavior
3. Navigate to the Widgets tab
4. Click on 'Add Widget'

**Expected behavior**
Clickable elements should be `<button>` tags

**Screenshots**

![image](https://user-images.githubusercontent.com/57726991/157947962-34c3ef9f-b997-465f-a436-02b51c389359.png)

**Comment by msnyder-msft** [comment]
fixed this one a while back but didn't close it out.
